README.md

```diff
@@ -0,0 +1,320 @@
+
+# Cat API #
+
+# A REST API using ExpressJS#
+
+A data repository to manage information about cats and breeds via the HTTP request-response cycle.
+
+# Getting Started #
+
+Run 'npm start' on Terminal.
```

Comment:
```suggestion
Run 'npm start' on Terminal, then follow the curl commands in the How to use section
```

It'd be good to hyperlink to the how to use section here so people can get straight to testing it out.